docs/v0.14/system-config.txt

```diff
@@ -2,4 +2,203 @@
 
 This article describes Fluentd's system configuraitons configured via ``<system>`` section and command line options.
 
-TBD
+## Overview
+
+System Configuration is one way to set up system-wide configuration such as enabling RPC, multiple workers, to obtain dump.
+
+## Parameters
+
+### workers (optional)
+
+Specify number of workers. Default is 1.
```

Comment:
This is same as other PR.
Could you use table instead of text for `type/default/version`?

Reply:
I updated to use table style for config_param parameters in system_config.rb.